centreon/www/front_src/src/Resources/Filter/criteriasNewInterface/basicFilter/atoms.ts

```diff
@@ -0,0 +1,13 @@
+import { atomWithStorage } from 'jotai/utils';
+import { atom } from 'jotai';
+
+import { SelectedResourceType } from '../model';
+
+export const selectedStatusByResourceTypeAtom =
+  atomWithStorage<Array<SelectedResourceType> | null>(
+    'centreon-23.10-resourceStatusRevamp-filterSelectedStatus',
```

Comment:
To do in another PR: We must find a way to get the set version using the platform version (23.10, 24.4) dynamically

Reply:
ook noté